crates/aptos-inspection-service/src/server/cpu_profiling.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright © Aptos Foundation
+
+use crate::server::utils::CONTENT_TYPE_JSON;
+use hyper::{Body, StatusCode};
+use std::{fs::File, path::Path, thread, time};
+
+pub fn handle_cpu_profiling_request() -> (StatusCode, Body, String) {
+    let guard = pprof::ProfilerGuard::new(100).unwrap();
+    let five_secs = time::Duration::from_millis(5000);
```

Comment:
does this mean we gather CPU sampling profile for 5 seconds? An option to run longer would be nice.

Reply:
5 secs were for demo purposes, but will either determine a more ideal duration or make it an input.